tests/ui/UnreadIndicatorsTest.js

```diff
@@ -23,6 +25,9 @@ import * as Pusher from '../../src/libs/Pusher/pusher';
 import PusherConnectionManager from '../../src/libs/PusherConnectionManager';
 import CONFIG from '../../src/CONFIG';
 
+// We need a large timeout here as we are lazy loading React Navigation screens and this test is running against the entire mounted App
+jest.setTimeout(30000);
```

Comment:
Makes sense to me from looking at the docs: https://jestjs.io/docs/jest-object#jestsettimeouttimeout